stable_horde.py

```diff
@@ -341,6 +344,11 @@ async def handle_request(self, req: Dict[str, Any]):
             self.state.sampler = sampler_name
             self.state.image = image
 
+            # Saving image locally
+            infotext = processing.create_infotext(p, p.all_prompts, p.all_seeds, p.all_subseeds, "Stable Horde", 0, 0) if shared.opts.enable_pnginfo else None
+            if self.config.save_images:
+                save_image(image, "horde", "", params['seed'], params['prompt'], "png", info=infotext, p=p)
```

Comment:
I think it would be better if the save path could be configured by the users. At least for me, I think it is not good to have an extra `horde` folder in my webui repo folder.

![image](https://user-images.githubusercontent.com/19927330/214071157-f5cf4631-c6d4-4bf9-b7dd-d04022fa13fc.png)

Reply:
Sure, I can add that to the config.
I'll do that in a day or so, if you don't.